lumibot/backtesting/polygon_backtesting.py

```diff
@@ -37,7 +37,7 @@ def __init__(
         # RESTClient API for Polygon.io polygon-api-client
         self.polygon_client = RESTClient(self._api_key)
 
-    def update_pandas_data(self, asset, quote, length, timestep, start_dt=None):
+    def update_pandas_data(self, asset, quote, length, timestep, start_dt=None, update_data_store=False):
```

Comment:
can you please add to the docstring details about "update_data_store" and what it does?

Reply:
let me know when you add this and i can merge

Reply:
Comment added.  I've also changed the name to _update_pandas_data to reflect its internal intent and will alert me if someone has a problem with that.